.github/workflows/ci.yml

```
@@ -9,14 +9,17 @@ on:
 jobs:
   build:
     runs-on: macos-latest
+    strategy:
+      matrix:
+        node: [14.19.0, 16.14.0]
```

You can just use the latest version instead of running on specific Node version
```suggestion
        node: [14, 16]
```

---

Good catch. I was wondering if it'd support it this way 👍